app/Http/Controllers/SubcriberController.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Subcriber;
+
+class SubcriberController extends Controller
+{
+    public function store(SubcribeRequest $request)
+    {
+        Subcriber::Create([
```

Comment:
```suggestion
        Subcriber::create([
```
hàm sao viết hoa thế a?